dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/CustomFormParam.test.tsx

```diff
@@ -32,11 +36,35 @@ afterAll((): void => {
 });
 
 it("should render a custom form component", () => {
-  const wrapper = mount(<CustomFormComponentLoader {...defaultProps} />);
+  const wrapper = mountWrapper(
+    getStore(defaultState),
+    <CustomFormComponentLoader {...defaultProps} />,
+  );
   expect(wrapper.find(CustomFormComponentLoader)).toExist();
 });
 
 it("should render the remote component", () => {
-  const wrapper = mount(<CustomFormComponentLoader {...defaultProps} />);
+  const wrapper = mountWrapper(
+    getStore(defaultState),
+    <CustomFormComponentLoader {...defaultProps} />,
+  );
   expect(wrapper.find(CustomComponent)).toExist();
 });
+
+it("should render the remote component with the default URL", () => {
+  const wrapper = mountWrapper(
+    getStore(defaultState),
+    <CustomFormComponentLoader {...defaultProps} />,
+  );
+  expect(wrapper.find(CustomComponent)).toExist();
```

Comment:
Should it also check the URL is the expected one (`....custom_components.js`) ?

Reply:
I will amend the test. Although it is assumed, an explicit test is a good idea 👍